pkg/corerp/frontend/controller/applications/graph_util.go

```diff
@@ -356,8 +361,8 @@ func computeGraph(applicationName string, applicationResources []generated.Gener
 		connectionsIn := connectionsByDestination[id]
 
 		entryConnections := make([]*corerpv20231001preview.ApplicationGraphConnection, len(connectionsIn))
-		for i, conn := range connectionsIn {
```

Comment:
This is the code bug. `&conn` refered to the same data so that entryConnections array held the same pointer values.

Reply:
Great catch!

Reply:
Ugh. 

The good news is that they are fixing this: https://github.com/golang/go/issues/60078

I wasn't able to find the link, but we can use the Go compiler to detect these cases.